docs/source/api/tests/index.rst

```diff
@@ -0,0 +1,16 @@
+Avocado Tests API Reference
+===========================
+
+At the most basic level there's the Test APIs which you should use when writing
+tests in Python and planning to make use of any other utility library.
+
+The Test APIs can be found in the :mod:`avocado` main module, and its most
+important member is the :class:`avocado.Test` class. By conforming to the
+:class:`avocado.Test` API, that is, by inheriting from it, you can use the full
+set of utility libraries.
+
+The Test APIs are guaranteed to be stable across a single major version of
+Avocado.  That means that a test written for a given version of Avocado should
```

Comment:
Duplicate spaces before "That."

Reply:
@willianrampazzo @clebergnu:

There are 592 double spaces after a dot on current documentation and the intention of this PR was to avoid any change in the content unless when it is conflicting with the new organization.

So, if you guys agree, I would suggest doing this change in the future on a different PR, if decided to go with one space.

Reply:
That is okay for me. Let's define a structure and change the content later.

Reply:
@beraldoleal agreed!